ProjectCars/Car.cs

```diff
@@ -18,11 +13,11 @@ public enum BrandCar
     }
     public class Car
     {
-        public string Id { get; set; }
-        public string Model { get; set; }
+        public String Id { get; set; }
```

Comment:
Two questions. Why do you change from string to String? And then, why the Id is a string?

Reply:
The Id is a string because I'm using the plate of the car like an id.
I saw String in some examples, but, I have been searching from your question, and I understand that is not the same string and String, and I must use string. Because string is a keyword from C#, and to use String I must put using System. Is that correct?

Reply:
About `string` vs `String` here is a fair response https://stackoverflow.com/a/7077. IMHO I prefer to use `string` in this class.

About type for an ID. It's a fair type if you want to use it as a plate.